serverjs/updatecards.js

```diff
@@ -596,6 +597,9 @@ function convertCard(card, isExtra) {
   newcard.scryfall_uri = card.scryfall_uri;
   newcard.rarity = card.rarity;
   newcard.oracle_text = card.oracle_text;
+  if (card.layout == 'adventure') {
+    newcard.oracle_text = card.card_faces[0].oracle_text + '\n' + card.card_faces[1].oracle_text;
+  }
```

Comment:
can we do this for all cards with multiple 'faces' objects?

Reply:
Yup, good idea (for searching purposes). Is the oracle text used for anything else that would muck up if we concatenate it?

Reply:
Nope, just used for filtering